main.go

```diff
@@ -76,8 +76,10 @@ func main() {
 	groupsBBAdapter := groups.NewGroupsAdapter(groupsHost, internalAPIKey)
 
 	//notifications BB adapter
+	ntAppID := getEnvKey("LMS_NOTIFICATIONS_APP_ID", true)
```

Comment:
Let's name them LMS_ORG_ID and LMS_APP_ID as they are not directly related with the notifications BB but are related with the multi tenancy support.